src/face_detector/launch/face_detector.launch

```diff
@@ -1,15 +1,18 @@
 <launch>
-	<node name = "camera" pkg = "usb_cam" type = "usb_cam_node">
-		<param name = "video_device" value = "/dev/video0" />
+	<node name="camera" pkg="usb_cam" type="usb_cam_node">
+		<param name="video_device"   value="/dev/video0" />
+		<param name="io_method"      value="mmap" />
+		<param name="pixel_format"   value="yuyv" />
```

Comment:
Excellent find, although I bet this means it won't work on my camera anymore. This might require a Snappy config solution.